ansible/roles/master/tasks/retrieve-systemd-files.yml

```diff
@@ -1,9 +1,6 @@
 ---
-- name: Get Systemd unit files from Kubernetes repository
-  get_url:
-    url=https://raw.githubusercontent.com/kubernetes/contrib/master/init/systemd/{{ item }}.service
```

Comment:
They are. However, the contrib repository is planned to be retired. So we need to find new place for all the systemd and config files.